common/config.go

```diff
@@ -3,35 +3,49 @@ package common
 import (
 	"fmt"
 	"os"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
 )
 
 var homeDir, _ = os.UserHomeDir()
 
 // Config contains constants that are defaults for various environments
 type Config struct {
-	Basepath             string
+	DataDir              string
 	MoneroDaemonEndpoint string
 	EthereumChainID      int64
-	Bootnodes            []string // TODO: when it's ready for users to test, add some bootnodes (#153)
+	ContractAddress      ethcommon.Address
+	Bootnodes            []string
 }
 
 // MainnetConfig is the mainnet ethereum and monero configuration
 var MainnetConfig = Config{
-	Basepath:             fmt.Sprintf("%s/.atomicswap/mainnet", homeDir),
+	DataDir:              fmt.Sprintf("%s/.atomicswap/mainnet", homeDir),
 	MoneroDaemonEndpoint: "http://127.0.0.1:18081/json_rpc",
 	EthereumChainID:      MainnetChainID,
 }
 
 // StagenetConfig is the monero stagenet and ethereum ropsten configuration
 var StagenetConfig = Config{
-	Basepath:             fmt.Sprintf("%s/.atomicswap/stagenet", homeDir),
+	DataDir:              fmt.Sprintf("%s/.atomicswap/stagenet", homeDir),
 	MoneroDaemonEndpoint: "http://127.0.0.1:38081/json_rpc",
-	EthereumChainID:      RopstenChainID,
+	EthereumChainID:      GorliChainID,
```

Comment:
Users no longer have to pass the chain ID on the command line when doing stagenet testing with Goerli.

Reply:
nice :D